### ISSUE TITLE
Proposal: Enable `global_namespace_import`

### ISSUE THREAD

**Issue by provokateurin** [nextcloud/coding-standard#36]
With configuration `['import_classes' => false, 'import_constants' => false, 'import_functions' => false]`: https://cs.symfony.com/doc/rules/import/global_namespace_import.html#example-3

Main benefit for me would be exceptions, as it is a directly clear that a docblock like
```php
/**
 * @throws Exception
 */
```
refers to `\OCP\DB\Exception` and not `\Exception`.
It would also safe some lines in the imports making them a bit more readable.

We could also switch it around, but I think we should settle on some configuration for this rule to have it the same everywhere.

**Comment by nickvergessen** [comment]
I agree that for global classes it's better readable/understandable if they are fully qualified inline

**Comment by blizzz** [comment]
I am fine either way
